x-pack/plugin/sql/qa/server/src/main/resources/datetime.csv-spec

```diff
@@ -153,6 +153,16 @@ SELECT WEEK(birth_date) week, birth_date FROM test_emp WHERE WEEK(birth_date) >
 2              |1953-01-07T00:00:00.000Z
 ;
 
+minuteOfDayFilterEquality
+SELECT MINUTE_OF_DAY(CONCAT(CONCAT('2021-01-22T14:26:06.', (salary % 2)::text), 'Z')::datetime) AS min_of_day 
```

Comment:
Not sure if adding the given time to the dates in `hire_date` would have been less contrived, but this works too.